CapabilityStatements/CapabilityStatement-mp-MedicationData.RetrieveServe.Base.xml

```diff
@@ -0,0 +1,392 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CapabilityStatement xmlns="http://hl7.org/fhir"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/R4/capabilitystatement.xsd">
+  <id value="mp-MedicationData.RetrieveServe.Base"/>
+  <url value="http://nictiz.nl/fhir/CapabilityStatement/mp-MedicationData.RetrieveServe.Base"/>
+  <version value="1.0.0" />
+  <name value="mp MedicationData.RetrieveServe.Base"/>
+  <status value="active"/>
+  <date value="2023-09-27"/>
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="url"/>
+      <value value="https://www.nictiz.nl"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="This CapabilityStatement describes the base requirements for a client/server to fulfill the 'Retrieve/Serve medication data' transaction within Medication Process 9 (MP9) version 3.0.0-beta.2. See [Landingspagina Medicatieproces](https://informatiestandaarden.nictiz.nl/wiki/Landingspagina_Medicatieproces) for additional information."/>
+  <purpose value="This CapabilityStatement is informative in nature and is designed as a base for the 'Retrieve/Serve medication data' transaction. For the MedMij context the mp-MedicationData.RetrieveServe.MedMij CapabilityStatement describes the minimal requirements, while the mp-MedicationData.RetrieveServe CapabilityStatement should be used in any other context."/>
```

Comment:
I think it's better to place the second sentence in `.description`. In profiles, we don't use `.purpose` either for these kinds of comments -- granted, we also don't use the `.description`, so maybe I'm basing this more on art than science.

I would also format the references to the other CapabilityStatements as Markdown links.

Reply:
Done!